Resources/Prototypes/Entities/Objects/Specific/Mech/mechs.yml

```diff
@@ -17,6 +17,17 @@
     - 7
     - 8
     #- 9 TODO: fusion
+    # remove everything except oxygen to maintain oxygen ratio
+    overflowGases:
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    - 6
+    - 7
+    - 8
+    #- 9 TODO: fusion
```

Comment:
Can't you just use the enum name here.

Reply:
other atmos prototypes like reactions dont so i assume not